pgx/src/datum/sql_entity_graph/pgx_sql.rs

```diff
@@ -983,35 +996,43 @@ fn connect_ords(
     for (item, &index) in ords {
         for (schema_item, &schema_index) in schemas {
             if item.module_path == schema_item.module_path {
-                tracing::debug!(from = ?item.full_path, to = schema_item.module_path, "Adding Ord after Schema edge.");
+                tracing::debug!(from = ?item.full_path, to = schema_item.module_path, "Adding Ord after Schema edge");
                 graph.add_edge(schema_index, index, SqlGraphRelationship::RequiredBy);
                 break;
             }
         }
         for (ty_item, &ty_index) in types {
             if ty_item.id_matches(&item.id) {
-                tracing::debug!(from = ?item.full_path, to = ty_item.full_path, "Adding Ord after Type edge.");
+                tracing::debug!(from = ?item.full_path, to = ty_item.full_path, "Adding Ord after Type edge");
                 graph.add_edge(ty_index, index, SqlGraphRelationship::RequiredBy);
                 break;
             }
         }
         for (ty_item, &ty_index) in enums {
             if ty_item.id_matches(&item.id) {
-                tracing::debug!(from = ?item.full_path, to = ty_item.full_path, "Adding Ord after Enum edge.");
+                tracing::debug!(from = ?item.full_path, to = ty_item.full_path, "Adding Ord after Enum edge");
                 graph.add_edge(ty_index, index, SqlGraphRelationship::RequiredBy);
                 break;
             }
         }
-        for (ty_item, &ty_index) in externs {
-            if ty_item.operator.is_some() {
-                tracing::debug!(from = ?item.full_path, to = ty_item.full_path, "Adding Hash after Operator edge.");
-                graph.add_edge(ty_index, index, SqlGraphRelationship::RequiredBy);
-                // NB:  no break here.  We need to be dependent on all externs that are operators
+        for (extern_item, &extern_index) in externs {
+            let fn_matches = |fn_name| item.module_path == extern_item.module_path && extern_item.name == fn_name;
+            let cmp_fn_matches = fn_matches(item.cmp_fn_name());
```

Comment:
This is pretty brittle and I'd prefer to extend this to be customizable in the future.